library/ssl_tls.c

```diff
@@ -4685,6 +4714,106 @@ const char *mbedtls_ssl_get_alpn_protocol( const mbedtls_ssl_context *ssl )
 }
 #endif /* MBEDTLS_SSL_ALPN */
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+static const mbedtls_ssl_srtp_profile_info srtp_profile_definitions[] =
```

Comment:
Design question, ping @mpg @gilles-peskine-arm: Do we want to export those strings unconditionally, even if they're for debugging purpose only? 

I prefer not do because it looks like a waste of ROM in constrained environments, but on the other hand, existing information structures, e.g. for Cipher or MD, do also contain a string representation of the respective object (which in the latter cases are much smaller, though).

Reply:
We've had complaints about too many strings values, although I'm not sure if it was about debugging code: the one I remember is that if you include anything that links to `oid.c`, you get all the OID. I think we should include them only if `MBEDTLS_DEBUG_C` is defined.

Reply:
Sounds good to me.